app/components/autocomplete/slash_suggestion/slash_suggestion_item.tsx

```diff
@@ -26,6 +29,14 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
             alignItems: 'center',
             marginTop: 8,
         },
+        httpIcon: {
+            width: 16,
+            height: 16,
+        },
+        dataIcon: {
+            width: 16,
+            height: 16,
```

Comment:
Does it make sense to have only one style?

Reply:
Good question. They are different entities, and I don't know if eventually we want to treat them differently. Probably not, since both are "normal" images, no matter if from data or http.

Will change to one single style.